test/python/primitive_test.py

```diff
@@ -249,3 +249,9 @@ def test_service():
         PingService().ping(nonce=u'nonce')
     with raises(TypeError):
         PingService().ping(wrongkwd=u'a')
+
+
+def test_optional_initializer_test():
```

Comment:
Since the function name already starts with `test_` we need no `_test` suffix here.